mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utils/TiltaksnavnUtils.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.mulighetsrommet.api.utils
+
+import no.nav.mulighetsrommet.api.domain.dto.TiltakstypeAdminDto
+
+class TiltaksnavnUtils {
+    companion object {
+        fun tilKonstruertNavn(tiltakstype: TiltakstypeAdminDto, arrangor: String?): String {
+            val casedTiltakstype = toTitleCase(tiltakstype.navn)
```

Comment:
Hvorfor toTitleCase på tiltakstype-navn? `toTitleCase` er vel kun for arrangørnavnet? :sweat_smile:

Reply:
Hvis man får tiltakstypenavn inn i small caps, men det er vel som regel ferdig kapitalisert uansett

Reply:
Tiltakstypenavnet er jo bestemt av oss/redaktører så vi burde ikke endre på det her